gradle.properties

```diff
@@ -12,8 +12,8 @@ cordaProductVersion = 5.1.0-INTEROP
 ##   a per module property in which case module versions can change independently.
 ## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
-## API version of last merge from corda mainline: 5
-cordaApiRevision = 9
+## API version of last merge from corda mainline: 2
```

Comment:
Why did this go backwards?

Reply:
I guess maybe it was lost during a merge etc?